### ISSUE TITLE
TypeError: Cannot read properties of null (reading 'concat')

### ISSUE THREAD

**Issue by vdesabou** [yamaton/vscode-h2o#8]
I was getting issue with sub-subcommand:

```
[Extension Host] [Completion] No completion item is available (1) TypeError: Cannot read properties of null (reading 'concat')
	at getOptions (/Users/vsaboulin/.vscode/extensions/tetradresearch.vscode-h2o-0.2.12/out/extension.js:558:40)
	at getCompletionsOptions (/Users/vsaboulin/.vscode/extensions/tetradresearch.vscode-h2o-0.2.12/out/extension.js:524:21)
	at Object.<anonymous> (/Users/vsaboulin/.vscode/extensions/tetradresearch.vscode-h2o-0.2.12/out/extension.js:66:47)
	at Generator.next (<anonymous>)
	at fulfilled (/Users/vsaboulin/.vscode/extensions/tetradresearch.vscode-h2o-0.2.12/out/extension.js:5:58)
```

I fixed it by changing:

```
const options = deepestCmd.options.concat(...inheritedOptionsArray);
```

to

```
const options = (!!deepestCmd && !!deepestCmd.options) ? deepestCmd.options.concat(...inheritedOptionsArray) : [];
```

It's probably not the best code, but that seems to work in my case

**Comment by yamaton** [comment]
Thank you so much for your report and the fix!
I've added it, and published as 0.2.13.

**Comment by vdesabou** [comment]
Works perfectly now !
